### PR TITLE
Clickable

### LINKED ISSUE #3: Not clickable
I am trying to access the site through https://isaacmsl.github.io/cafe.music/
Webpage is not clickable.

Session Info:
Brave Browser in mac

### PR DESCRIPTION
**Pull request by st-prasad**
my contribution for the issue[#3](https://github.com/isaacmsl/cafe.music/issues/3). Added clickable buttons
Closes #3

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,9 +9,48 @@
     <link rel="stylesheet" href="style.css">
     <script type="text/javascript" src="radios.js" defer></script>
     <script type="text/javascript" src="script.js" defer></script>
+
+    <style>
```
Comment (isaacmsl) [inline index.html:13]: Please transfer this style to `style.css`

File: script.js [hunk 2]
```diff
@@ -60,5 +60,17 @@ const checkKey = (event) => {
     }
 }
 
+    // btn onclick events
```
Comment (isaacmsl) [inline script.js:63]: Please fix indentation. By the way, prefer using `() => {}` instead `function(){}`.

**Review by isaacmsl — CHANGES_REQUESTED** [review]
Nice! Can you make those changes? :rocket:

File: script.js (hunk 2 above)
Comment (st-prasad) [reply script.js:63]: ok, will update

**Review by isaacmsl — APPROVED** [review]
LGTM 🚀

**Pull request merged**